src/main/java/de/paul2708/memory/command/MemoryCommand.java

```diff
@@ -16,31 +16,31 @@
     @Override
     public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
 
-        if (sender instanceof Player) {
-            Player player = (Player) sender;
-            Queue queue = Memory.getGameManager().getQueue();
-
-            if (queue.contains(player)) {
-                queue.remove(player);
-                player.sendMessage(Memory.getMessageFile().getMessage("queue.leave"));
-            } else {
-                queue.add(player);
+        if (!(sender instanceof Player)) {
+            Memory.getInstance().log(Constants.TAG + "§cYou cannot run this command from console.");
+            return false;
```

Comment:
It should return `true`, otherwise, the command usage will be sent to the console as well.

Reply:
You didn't specify any usage so no usage will be send.